src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -940,9 +1000,34 @@ static void window_cheats_paint(rct_window *w, rct_drawpixelinfo *dpi)
         gfx_draw_string_left(dpi, STR_BOTTOM_TOOLBAR_CASH, gCommonFormatArgs, colour, w->x + XPL(0) + TXTO, w->y + YPL(2) + TXTO);
     }
     else if(w->page == WINDOW_CHEATS_PAGE_MISC){
+        sint32 monthstr = 0;
+        switch (month_spinner_value) {
+        case 1: monthstr = STR_MONTH_MARCH;
```

Comment:
This is basically `DateGameMonthNames` in `localisation.c`. It's a bit of a waste to duplicate it here…

You can leverage it by making a new string `STR_FORMAT_MONTH` that uses the `{MONTH}` formatting code to use in your call to `gfx_draw_string_right` below. The argument can be passed using `set_format_arg(offset, type, value)`.